content/en-us/tutorials/fundamentals/coding-5/making-changes-to-arrays.md

```diff
@@ -91,7 +91,11 @@ Notice also that all the index values changed or have been pushed. Removing a va
 
 ## Searching for Values in an Array
 
-There's no built-in function to search an array. To find specific values in arrays, like the name of a winning player, you'll need to code your own **search function** using for loops and if statements.
+To find specific values in arrays, like the name of a winning player, you can code your own **search function** using for loops and if statements.
+
+<Alert severity="info">
+There is already a builtin function for this: `table.find`
+</Alert>
```

Comment:
```suggestion
To find specific values in arrays, like the name of a winning player, use the `Library.table.find()` function. Alternatively, you can code your own search function using `for` loops and `if` statements.
```

Reply:
Seems like table.find() is just much more straightforward than the alternative, so putting that first.

Also for your reference (if you haven't found it already), see https://github.com/Roblox/creator-docs/blob/main/STYLE.md#link-syntax for how API page links can be crafted within backticks as I've done for table.find() here.